### PR TITLE
ximgproc: add python wrappers for disparity functions

### PR DESCRIPTION
**Pull request by berak**
### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [x] I agree to contribute to the project under OpenCV (BSD) License.
- [x] To the best of my knowledge, the proposed patch is not based on a code under GPL or other license that is incompatible with OpenCV
- [x] The PR is proposed to proper branch
- [ ] There is reference to original bug report and related work
- [ ] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [ ] The feature is well documented and sample code can be built with the project CMake

questions:
* should the license be changed ?
* does it need a python test ? (there is already a [c++ one here](https://github.com/opencv/opencv_contrib/blob/master/modules/ximgproc/test/test_disparity_wls_filter.cpp#L19))

### CODE REVIEW

**Comment by alalek** [comment]
3.4 code is under BSD license. 4.5+ is under Apache 2.0 license with BSD code snippets.
Python test should check binding interface only (input type, output type and output size is enough). No need to validate function accuracy.

**Comment by berak** [comment]
> 3.4 code is under BSD license.

right, i forgot ;)

File: modules/ximgproc/misc/python/test/test_disparity.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import cv2 as cv
+import numpy as np
+
+from tests_common import NewOpenCVTests
+
+class disparity_test(NewOpenCVTests):
+    def test_disp(self):
+        # readGT
+        ret,GT = cv.ximgproc.readGT(self.find_file("cv/disparityfilter/GT.png"))
+        self.assertFalse(ret) # returns 0 on success!
```
Comment (alalek) [inline modules/ximgproc/misc/python/test/test_disparity.py:11]:
```
self.assertEqual(ret, 0)
```
?

**Review by alalek — APPROVED** [review]
Well done! Thank you :+1:

**Pull request merged**